homeassistant/components/onvif/parsers.py

```diff
@@ -10,6 +11,15 @@
 PARSERS: Registry[str, Callable[[str, Any], Coroutine[Any, Any, Event]]] = Registry()
 
 
+def datetime_or_zero(value: str):
```

Comment:
Please add a type annotation of the return value.

Reply:
Done